action/remove.go

```diff
@@ -18,10 +18,7 @@ var kubeGet kubeGetter = func(m string) string {
 	log.Debug("Getting manifests from %s", m)
 
 	a := []string{"get", "-f", m}
-	out, err := exec.Command("kubectl", a...).CombinedOutput()
-	if err != nil {
-		log.Err("Could not get manifest. %s", err)
-	}
+	out, _ := exec.Command("kubectl", a...).CombinedOutput()
```

Comment:
@arschles `out` is not empty on error.  the error that gets returned is `exit status 1` which is not useful for the user.